src/Evolution/Systems/GeneralizedHarmonic/GhDampingFunctions/GaussianPlusConstant.hpp

```diff
@@ -0,0 +1,123 @@
+#include "Options/Options.hpp"
+#include "Parallel/CharmPupable.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class DataVector;
+/// \endcond
+
+namespace GhDampingFunctions {
+template <size_t VolumeDim, typename Fr>
+class GaussianPlusConstant;
+
+/*!
+ * \ingroup GhDampingFunctionsGroup
+ * \brief GaussianPlusConstant \f$f = C + A
+ * \exp\left(-\frac{(x-x_0)^2}{w^2}\right)\f$
+ *
+ * \details Input file options are: `Constant` \f$C\f$, `Amplitude` \f$A\f$,
+ * `Width` \f$w\f$, and `Center`\f$x_0\f$. The function takes input coordinates
+ * of type `tnsr::I<T, VolumeDim, Fr>`, where `T` is e.g. `double` or
+ * `DataVector`, `Fr` is a frame (e.g. `Frame::Inertial`), and `VolumeDim` is
+ * the dimension of the spatial volume.
+ */
+template <size_t VolumeDim, typename Fr>
+class GaussianPlusConstant : public GhDampingFunction<VolumeDim, Fr> {
+ public:
+  struct Constant {
+    using type = double;
+    static constexpr OptionString help = {"The constant."};
+  };
+
+  struct Amplitude {
+    using type = double;
+    static constexpr OptionString help = {"The amplitude."};
+  };
+
+  struct Width {
+    using type = double;
+    static constexpr OptionString help = {"The width."};
+    static type lower_bound() noexcept { return 0.; }
+  };
+
+  struct Center {
+    using type = std::array<double, VolumeDim>;
+    static constexpr OptionString help = {"The center."};
+  };
+  using options = tmpl::list<Constant, Amplitude, Width, Center>;
+
+  static constexpr OptionString help = {
+      "Computes a Gaussian plus a constant about an arbitrary coordinate "
+      "center with given width and amplitude"};
+
+  WRAPPED_PUPable_decl_base_template(
+      SINGLE_ARG(GhDampingFunction<VolumeDim, Fr>),
+      GaussianPlusConstant);  // NOLINT
+
+  explicit GaussianPlusConstant(CkMigrateMessage* /*unused*/) noexcept {}
+
+  GaussianPlusConstant(double constant, double amplitude, double width,
+                       const std::array<double, VolumeDim>& center) noexcept;
+
+  GaussianPlusConstant() = default;
+  ~GaussianPlusConstant() override = default;
+  GaussianPlusConstant(const GaussianPlusConstant& /*rhs*/) = delete;
+  GaussianPlusConstant& operator=(const GaussianPlusConstant& /*rhs*/) = delete;
+  GaussianPlusConstant(GaussianPlusConstant&& /*rhs*/) noexcept = default;
+  GaussianPlusConstant& operator=(GaussianPlusConstant&& /*rhs*/) noexcept =
+      default;
+
+  Scalar<double> operator()(
+      const tnsr::I<double, VolumeDim, Fr>& x) const noexcept override;
+  Scalar<DataVector> operator()(
+      const tnsr::I<DataVector, VolumeDim, Fr>& x) const noexcept override;
+
+  // clang-tidy: google-runtime-references
+  void pup(PUP::er& p) override;  // NOLINT
+
+ private:
+  double constant_{};
+  double amplitude_{};
+  double inverse_width_{};
+  std::array<double, VolumeDim> center_{};
+  friend bool operator==(const GaussianPlusConstant& lhs,
```

Comment:
Could you move the friend function before the member variables and add a blank line between them? Not a strict rule, but I find it easier to locate member variables if friend functions aren't at the end of the class. E.g.

- friend functions
- private member functions
- member variables

If you prefer your current ordering, could you please add a blank line between the friend function and the member variables?